styles.css

```diff
@@ -0,0 +1,101 @@
+@import url(./css/utilities/variables.css);
+@import url(./css/layouts/navbar.css);
+@import url(./css/layouts/footer.css);
+html{
+    background-color: var(--lightest-green);
+    font-family: var(--acme-font);
+}
+
+/* Header */
+.hero-header{
+    width: 100%;
+    height: 80vh;
+    color: var(--secondary);
+    background-image: url(./assets/Images/farm.jpg);
+    background-repeat:no-repeat;
+    background-size:cover;
+    background-position: center;
+    position: relative;
+}
+
+.banner-title{
+    margin-top: 1.2rem;
+}
+
+.btn-shop-now{
+    position: absolute;
+    background-color: white;
+    border: 2px solid var(--primary-hover);
+    color: var(--primary);
+    width: 10rem;
+    font-size: 1.5rem;
+    padding:0.5rem 1.5rem;
+    top: 50%;
+    left: 50%;
+    /* Center the button in div */
+  -ms-transform: translate(-50%, -50%);
+  transform: translate(-50%, -50%);
+}
+
+.btn-shop-now:hover{
+    color: white;
+    border: 2px solid var(--white);
+}
+.title-underline {
+    background-color: var(--primary);
+    width: 10rem;
+    height: 0.25rem;
+    margin: 0.5rem auto 2rem;
+}
+.categories{
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(5rem, 15rem));
```

Comment:
just a suggesstion , you can also use flex and give flex-wrap for responsiveness, as grid is mostly used for the overall layout of the page.

Reply:
Thanks for your Review, I am planning to add some more categories in future so thought of grid will be best to use.